### PR TITLE
[PLAY-8] Added Border Radius Tokens to Visual Guidelines PB

### PR DESCRIPTION
**Pull request by RexGluck**
#### Screens

[INSERT SCREENSHOT]
Screen Shot 2021-11-18 at 10 16 36 AM

#### Breaking Changes

[Yes/No (Explain)]

#### Runway Ticket URL

[https://nitro.powerhrg.com/runway/backlog_items/PLAY-8]

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 1]
```diff
@@ -42,4 +45,68 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
-}
\ No newline at end of file
+
+
+#border_radius_container{
+  margin: 0px 10px 0px 0px !important;
```
Comment (kre8sions) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:51]: Let's use our spacing variables or we can use the margin classes on the actual div themselves and delete it from here.

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 2]
```diff
@@ -42,4 +45,68 @@
...
+  text-align: center;
+}
+
+#border_radius_rounded{
+  border: $border_light 1px solid !important;
+  height: 100px !important;
```
Comment (kre8sions) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:57]: Let's remove all !important tags

Comment (kre8sions) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:57]: @RexGluck

**Review by kre8sions — CHANGES_REQUESTED** [review]
Image 2021-11-17 at 8 39 51 AM

**Comment by jasoncypret** [comment]
It's looking great. Let's just get some microscopy added!!

Border Radius

**Comment by jasoncypret** [comment]
@RexGluck you've got conflicts.

```
git checkout master
git pull
git checkout [yourbranch]
git merge origin/master
git push --force-with-lease
```

If you get conflicts just accept all incoming.

**Comment by jasoncypret** [comment]
LGTM 🚀

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 3]
```diff
@@ -41,4 +44,61 @@
...
+    margin: 20px 15px 15px;
+    border-radius: $border_radius_xs;
+  }
+
+  #border_radius_none{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_none;
+  }
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:101]: In order to keep this file clean lest move this into a partial and import it here.

Also you need a space after your selector and your curly brace `.selector {`

File: playbook-website/app/views/pages/visual_guidelines_partials/_border_radius.html.erb [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+        <div class="pr_sm" id="border_radius_container">
+            <div class="m_sm" id="border_radius_none"></div>
+            <%= pb_rails("title", props: { text: "None", tag: "h4", size: 4 }) %>
+            <%= pb_rails("caption", props: { text: "$border_radius_none", size: 'xs' }) %>
+        </div>
+        
+    <% end %>
+    
+    <% react_contents = Rails.root.join("app/views/pages/code_snippets/border_radius_tokens.txt").read %>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_border_radius.html.erb:55]:
```suggestion
    <% border_radius_tokens = Rails.root.join("app/views/pages/code_snippets/border_radius_tokens.txt").read %>
```

We will probably have another story to move these to the controller. In the meantime, let's rename some this does not collide with anyone else.

**Review by jasoncypret — CHANGES_REQUESTED** [review]
Just a couple of tweaks. Remind me to show you some fancy sass selector magic. 😉

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**